src/ui/mentionsview.js

```diff
@@ -27,7 +32,7 @@ export default class MentionsView extends View {
 			attributes: {
 				class: [
 					'ck',
-					'ck-mention'
+					'ck-mention-view'
```

Comment:
I guess you wanted to change this because it collides (or will in the future) with a single mention in the content of the editor?

Whatever the reason, this naming convention is incorrect. All UI Views are... views, so there's no point in keeping the `-view` suffix in the class name (and we don't do that). The correct class would be `ck-mentions` or `ck-mention-list` or `ck-mentions-wrapper`, etc. (I'm not 100% sure about the role of this class in the UI).